server/apps/immich/src/controllers/auth.controller.ts

```diff
@@ -31,8 +31,8 @@ export class AuthController {
     @Req() req: Request,
     @Res({ passthrough: true }) res: Response,
   ): Promise<LoginResponseDto> {
-    const { response, cookie } = await this.authService.login(loginCredential, clientIp, req.secure);
-    res.setHeader('Set-Cookie', cookie);
```

Comment:
What is the different between the two APIs?

Reply:
`set(key, value)`, `setHeader(key, value)`, and `header(key, value)` all do the same thing as far as I know.

`set` and `header` are methods from express, `setHeader` is a method from the node `http` standard library.